### DISCUSSION TITLE
can you official launch width percentage like this

### DISCUSSION THREAD

**Discussion by HOTLancerX** [tailwindlabs/tailwindcss#14157] [General]
0 to 100 %
wp-1 //1%
wp-2 //2%
etc..
wp-33.3333 //33.3333%
etc..
wp-100 //100%

**Comment by franktucker907** [comment]
Hello,
It looks like you’re trying to create a list that maps percentages to specific waypoints (wp). Here’s a structured format for you
[SecureSpend card](https://www.securespend.com.co)
wp-1 // 1%
wp-2 // 2%
wp-3 // 3%
...
wp-33 // 33%
...
wp-100 // 100%
Best Regards
franktucker907

**Comment by HOTLancerX** [comment]
(wp) = width percentage
